.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,39 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/docker-existing-dockerfile
+{
+	"name": "Existing Dockerfile",
+	"build": {
+		// Sets the run context to one level up instead of the .devcontainer folder.
+		"context": "..",
+		// Update the 'dockerFile' property if you aren't using the standard 'Dockerfile' filename.
+		"dockerfile": "../Dockerfile.dev",
+		"cacheFrom": [
+			"type=registry,ref=ghcr.io/aarhus-psychiatry-research/psycop-common:latest"
```

Comment:
@KennethEnevoldsen You might find this a bit sexy.

Reply:
Hmmm tror jeg skal have en intro til hvad der gør

Reply:
Når vi bygger en dev container lokalt, bygger den udfra dockerfilen.

Nu bliver den bygget hver gang vi laver et push til main, og gemt i et container repository. Hvis jeg vil starte en ny dev container lokalt, cacher den fra remote container repositoriet.

I.e., meget hurtigere dev environment setups!